.github/workflows/master_only.yml

```diff
@@ -134,3 +134,43 @@ jobs:
         run: pytest --verbose --color=yes sdk/python/tests --integration --benchmark --benchmark-autosave --benchmark-save-data --durations=5
       - name: Upload Benchmark Artifact to S3
         run: aws s3 cp --recursive .benchmarks s3://feast-ci-pytest-benchmarks
+
+  build-publish-docker-images:
```

Comment:
Why do we need to publish these images for every commit?

Reply:
Removed as discussed offline. Keeping the build though for a continuous build